### ISSUE TITLE
Enhanced support for screen reader users

### ISSUE THREAD

**Issue by SamKacer** [DonJayamanne/vscodeJupyter#136]
Hello,

good job on the extension. I like it a lot. 

My main reason for using it is because I am blind and so use a screen reader for feedback instead of a screen. Jupyter notebook has very poor accessibility because it uses the CodeMirror editor which is entirely inaccessible. However editing/running jupyter notebooks in VS code with this extension is a pretty good accessible alternative.

One thing that would greatly enhance the accessibility of this extension and greatly improve the workflow is if the screen reader user could access the results for output immediatly. Current issue is that when I would like to read the output I have to move my focus of the screen reader from the editor to the results pane and then when i want to do more editions or run some cells, etc. I have to move the focus back. This moving of focus back and forth slows down the workflow, although the workflow is still much better than using jupyter notebook directly.

Anyway, I have two ideas that might solve this problem for screen reader users.

First one is this:
VS code GUI is made in Electron if I am not mistaken, which has the elements essentially in html. the attribute 'aria-live="polite"' could be applied to  the html element that contains the output text. If aria-live works correctly in electron, then if the content changes, then the changes should be read by the screen reader.

The other option if this doesnt work is creating a shortcut, which moves the system focus to the area with the text or to the clear results button, which precedes the output, so that screen reader users have a more convenient shortcut for navigating to the results pane.

Regards
